CONTRIBUTING.md

```diff
@@ -70,6 +76,28 @@ docker compose run --rm node_shell
     docker compose run --rm mkdocs_build_site
     ```
 
+#### Playwright
+
+npm commands such as `test:playwright-ct:all` and `test:playwright-ct:all-with-update`
+need to be run them within the `playwright` Docker container.
+
+To log into the container, run:
+
+```bash
+docker compose run --rm playwright
+```
+
+If you want to run single command, run:
+
+```bash
+docker compose run --rm playwright bash -c 'npm run <command>'
```

Comment:
If we wanrt to have this detail, I would write:

`docker compose run --rm playwright bash -c 'npm run test:playwright-ct:*'`

To illustrate, that it is not correct to run anything else there.